<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListagemProdutos</class>
 <widget class="QDialog" name="ListagemProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listagem de Produtos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="logoHorizontalLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="minimumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/imagens/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cadastroLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCadastrarProduto">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton {
          background-color: #FF66B2;
          color: white;
          border-radius: 15px;
          border: 2px solid #FF3399;
          padding: 10px 20px;
          font-size: 16px;
         }
         QPushButton:hover {
          background-color: #FF3399;
          border-color: #FF66B2;
         }
        </string>
       </property>
       <property name="text">
        <string>Cadastrar Novo Produto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filtrosLayout">
     <item>
      <widget class="QLineEdit" name="buscaLineEdit">
       <property name="styleSheet">
        <string>
         QLineEdit {
          border: 2px solid #FF66B2;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
          color: #FF66B2;
         }
         QLineEdit:focus {
          border: 2px solid #FF3399;
         }
        </string>
       </property>
       <property name="placeholderText">
        <string>Buscar por Nome, Marca ou Código de Barras...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="grupoComboBox">
       <property name="styleSheet">
        <string>
         QComboBox {
          border: 2px solid #FF66B2;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
          color: #FF66B2;
         }
         QComboBox:focus {
          border: 2px solid #FF3399;
         }
         QComboBox::drop-down {
          border: none;
          background: transparent;
         }
         QComboBox::down-arrow {
          image: url(:/resources/arrow-down.png);
         }
        </string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelaProdutos">
     <property name="columnCount">
      <number>9</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grupo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Qtd</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preço</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botoesLayout">
     <item>
      <widget class="QPushButton" name="btnVoltar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton {
          background-color: transparent;
          color: #DC3545;
          border-radius: 15px;
          border: 2px solid #DC3545;
          padding: 8px 15px;
          font-size: 16px;
         }
         QPushButton:hover {
          background-color: #DC3545;
          color: white;
          border: 2px solid #DC3545;
         }
        </string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
